tests/requirements.txt

```diff
@@ -1,23 +1,24 @@
 anaconda-client
 beautifulsoup4
 chardet
-conda >=4.13
+conda >=22.11.0
 conda-forge::allure-pytest
 conda-index
-conda-package-handling
+conda-package-handling >=1.3
 conda-verify
 contextlib2
 cytoolz
 filelock
 git
 jinja2
-jsonschema
+jsonschema >=4.19
+menuinst
 numpy
+packaging
 perl
 pip
 pkginfo
 psutil
-py-lief
```

Comment:
Why remove this for Windows?

Reply:
because thats what we do in our recipe 🤷🏻‍♂️ 

https://github.com/conda/conda-build/blob/328f7e1f9939125efcc934eb5f2c44343b303208/recipe/meta.yaml#L45

Reply:
actually removing this appears to have broken the overlinking test? adding back